src/motor/motor_manager.cpp

```diff
@@ -186,7 +190,11 @@ std::vector<std::shared_ptr<Motor>> MotorManager::get_motors_by_ip(std::vector<s
             m[j++] = motor;
         } else {
             if (print_unconnected) {
-                std::cerr << "Motor IP: " << motor->addrstr_ << "(" << motor->hostname_ << ") not connected" << std::endl;
+                std::cerr << "Motor IP: " << motor->addrstr_ << "(" << motor->hostname_;
+                if (motor->ip_alias_.size()) {
+                    std::cerr << ": " << motor->ip_alias_;
+                }
+                std::cerr << ") not connected" << std::endl;
```

Comment:
Or maybe I'm confused. Is ip_alias new? Does it not come from the json file?

Reply:
It will print both. I found it's actually not always obvious whether you're going to the ip or the text name from getnameinfo() (which provides hostname_), so it could give an ip the then name also, or if someone doesn't have it /etc/hosts then you'll get the ip twice followed by the json name. Or sometimes the ip and the ascii name flip. It's weird. 
```Motor IP: 192.168.50.10(192.168.50.10: test_actuator) not connected```
or you could get
```Motor IP: 192.168.50.10(test_actuator: test_actuator) not connected```
but in some cases
```Motor IP: 192.168.50.10(test_actuator: test_actuator_j1) not connected```
Anyway yes, it's a little strange that I'm passing an extra string down to MotorIP. The reason is that I'm trying to maintain a license-less distribution. So you can run MotorIP on a robot without giving credit to nlohmann. But you would if you want to run motor_util.

Reply:
Another downside is you don't get the device_ip_map.json in python right now. So I would probably have to re-parse it there or not worry about the license.

Reply:
It might get confusing in practice. If it does, we might be able to simplify things. For example, if the hostname is the same as the IP (i.e. an entry is not present in /etc/hosts), but different from the alias, we probably don't need to print the IP twice:
`Motor IP: 192.168.50.10(alias: test_actuator) not connected`